design/Limits.md

```diff
@@ -14,16 +14,22 @@ dictionary GPULimits {
     unsigned long maxStorageBuffersPerShaderStage = 4;
     unsigned long maxStorageTexturesPerShaderStage = 4;
     unsigned long maxUniformBuffersPerShaderStage = 12;
+    unsigned long maxVertexBuffers = 8;
+    unsigned long maxVertexAttributes = 16;
+    unsigned long maxVertexArrayStride = 2048;
```

Comment:
I'm not sure if `maxVertexArrayStride` is the name we want, but the editors can follow-up on that separately, no need to block this PR.

Reply:
Should I create a new issue for this?

Reply:
We can file an issue during the Monday editing meeting if we don't immediately decide on something.